src/map/packet_system.cpp

```diff
@@ -4142,6 +4142,11 @@ void SmallPacket0x085(map_session_data_t* const PSession, CCharEntity* const PCh
 void SmallPacket0x096(map_session_data_t* const PSession, CCharEntity* const PChar, CBasicPacket data)
 {
     TracyZoneScoped;
+    if (!PTarget)
```

Comment:
PTarget doesn't exist yet here, you likely want this check immediately after the line:
```cpp
 CCharEntity* PTarget = (CCharEntity*)PChar->GetEntity(PChar->TradePending.targid, TYPE_PC);
 ```

Reply:
Yikes. Corrected.